src/rfxencode_dwt_shift_rem.c

```diff
@@ -44,7 +44,7 @@
 
 #define SETUPLOQ(_index, _shift) do { \
     lo_fact = (((quants[_index] >> (_shift)) & 0xf) - 6) + DWT_FACTOR; \
-    lo_half = 1 << (hi_fact - 1); } while (0)
+    lo_half = 1 << (lo_fact - 1); } while (0)
```

Comment:
I remember copying this code from your branch and being confused as to why it was hi_fact here :)

Reply:
oh, hehe, well your confusion was warranted.